test/functional/test_framework/mininode.py

```diff
@@ -236,6 +236,8 @@ def __init__(self):
         # The network services received from the peer
         self.nServices = 0
 
+        self.timeout = time_to_connect * 5
```

Comment:
Why are you multiplying by 5 here?

Reply:
5 is just a random number, there is no reason behind it. I want to make the timeout long enough to wait for network and wallet to sync

Reply:
can you  add this intention as a comment?